docs/sources/dashboards/manage-dashboards/index.md

```diff
@@ -71,21 +71,21 @@ You can complete the following tasks on the **Dashboard Folder** page:
 - Rename a folder (available under the **Settings** tab)
 - Assign permissions to folders (which are inherited by the dashboards in the folder)
 
-To navigate to the dashboard folder page, click the cog appears when you hover over a folder in the dashboard search result list or the **Manage dashboards and folders** page.
+To navigate to the dashboard folder page, click the cog that appears when you hover over a folder in the dashboard search result list or the **Manage dashboards and folders** page.
 
 ### Dashboard permissions
 
-You can assign permissions to a folder. Any permissions you assign are inherited by the dashboards in the folder. An Access Control List (ACL) is used where **Organization Role**, **Team** and a **User** can be assigned permissions.
+You can assign permissions to a folder that are inherited by the dashboards in the folder. An Access Control List (ACL) is used where **Organization Role**, **Team** and a **User** can be assigned permissions.
 
 For more information about dashboard permissions, refer to [Dashboard permissions]({{< relref "../../administration/roles-and-permissions/#dashboard-permissions" >}}).
 
 ## Export and import dashboards
 
-You can use the Grafana UI or the [HTTP API]({{< relref "../../developers/http_api/dashboard/#create-update-dashboard" >}}) to export and import dashboards.
+You can use the Grafana UI or the [HTTP API]({{< relref "../../developers/http_api/dashboard/#create-update-dashboard" >}}) to export and import dashboards. Grafana dashboards can also be [managed as code[(https://grafana.com/blog/2022/12/06/a-complete-guide-to-managing-grafana-as-code-tools-tips-and-tricks/), using tools such as Terraform.
 
 ### Export a dashboard
 
-The dashboard export action creates a Grafana JSON file that contains everything you need, including layout, variables, styles, data sources, queries, and so on, so that you can later import the dashboard.
+The dashboard export action creates a Grafana JSON file that contains everything within the dashboard including layout, variables, styles, data sources, queries, etc. which can be used to import the dashboard.
```

Comment:
Your edit on the last part of the sentence changes the subject of that clause. Instead of all of those things being included in the the JSON file that you can now import if you want to, you've got all these individual elements that can be used to import the dashboard, which isn't really what we mean. Recommend leaving that last chunk as is but using different punctuation (dashes): 
```suggestion
The dashboard export action creates a Grafana JSON file that contains everything within the dashboard—including layout, variables, styles, data sources, queries, and so on—so that you can later import the dashboard.
```